tests/comm/test_pcie_dcp_topk_lifecycle.py

```
@@ -0,0 +1,758 @@
+    cp = _CP(ws=2)
+    _patch(monkeypatch, cp)
+    c0, _ = _rank(monkeypatch, cp, 0)
+    c1, _ = _rank(monkeypatch, cp, 1)
+    errs: list[Exception | None] = []
+
+    def close_twice():
+        def once():
+            try:
+                c0.close()
+                errs.append(None)
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)